viz.Rmd

```diff
@@ -547,16 +788,26 @@ are blending together, and although it is possible to derive *some* insight from
 of the most incorrect measurements), it isn't the clearest way to convey our message and answer the question. Let's try a different strategy of 
 creating multiple separate histograms on top of one another. 
 
-In order to create a plot in `ggplot2` that has multiple subplots arranged in a grid, we use the `facet_grid` function.
-The argument to `facet_grid` specifies the variable(s) used to split the plot into subplots. It has the syntax `vertical_variable ~ horizontal_variable`,
-where `veritcal_variable` is used to split the plot vertically, `horizontal_variable` is used to split horizontally, and `.` is used if there should be no split
-along that axis. In our case, we only want to split vertically along the `Expt` variable, so we use `Expt ~ .` as the argument to `facet_grid`.
+In order to create a plot in `ggplot2` 
+that has multiple subplots arranged in a grid, 
+we use the `facet_grid` function.
+The argument to `facet_grid` specifies the variable(s) used to split the plot 
+into subplots, and how to split them (i.e., into rows or columns).
+If the plot is to be split horizontally, into rows, 
+then the `rows` argument is used.
+If the plot is to be split vertically, into columns, 
+then the `columns` argument is used.
+Both the `rows` and `columns` argument take the column names to split the data 
+on when creating the subplots. 
+One key thing is that the column names must be surrounded by the `vars` function.
+This function allows the column names to be correctly evaluated 
+in the context of the data frame.
 \index{ggplot!facet\_grid}
 
-```{r 03-data-morley-hist-4, warning=FALSE, message=FALSE, fig.cap = "Histogram of Michelson's speed of light data split vertically by experiment"}
-morley_hist <- ggplot(morley, aes(x = Speed, fill = factor(Expt))) +
+```{r 03-data-morley-hist-4, warning=FALSE, message=FALSE, fig.height = 7, fig.cap = "Histogram of Michelson's speed of light data split vertically by experiment"}
+morley_hist <- ggplot(morley, aes(x = Speed, fill = as_factor(Expt))) +
   geom_histogram(position = "identity") +
-  facet_grid(Expt ~ .) +
+  facet_grid(rows = vars(Expt)) +
   geom_vline(xintercept = 792.458, linetype = "dashed", size = 1.0)
 morley_hist
 ```
```

Comment:
I think after the line: "makes it quite clear how accurate the different experiments were with respect to one another. "
I think it would help the reader to provide a quick interpretation of the plot e.g. "We can experiment 1 had the highest spread/variability (whatever term we have used in the past) and experiment 6 blah etc."